sandy_sort_lists.py

```diff
@@ -0,0 +1,94 @@
+import config
+import requests
+import json
+
+
+COLOR_KEYS = {
+	"green": 1,
+	"yellow": 2,
+	"orange": 3,
+	"red": 4,
+	"purple": 5,
+	"blue": 6,
+	"sky": 7,
+	"lime": 8,
+	"pink": 9,
+	"black": 10
+}
+BOARD_IDS_URL = "https://api.trello.com/1/members/{username}/boards"
+GET_LISTS_URL = "https://api.trello.com/1/boards/{board_id}/lists"
+GET_CARDS_URL = "https://api.trello.com/1/lists/{list_id}/cards"
+PUT_CARDS_URL = "https://api.trello.com/1/cards/{card_id}/pos"
+BASIC_PARAMS = {
```

Comment:
This fine for this script.

But there are advantages to putting this in a class. E.g. each class can have it's own credentials. And then we can make Trello requests on behalf of multiple users from the same app if we needed to.